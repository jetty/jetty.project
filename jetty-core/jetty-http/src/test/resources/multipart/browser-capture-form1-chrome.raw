------WebKitFormBoundary46EP6zTN86hbbaJC
Content-Disposition: form-data; name="user"

joe
------WebKitFormBoundary46EP6zTN86hbbaJC
Content-Disposition: form-data; name="comment"

this is a simple comment
------WebKitFormBoundary46EP6zTN86hbbaJC--
