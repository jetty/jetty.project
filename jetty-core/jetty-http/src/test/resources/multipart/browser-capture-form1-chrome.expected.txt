Request-Header|Accept|text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8
Request-Header|Accept-Encoding|gzip, deflate, br
Request-Header|Accept-Language|en-US,en;q=0.9
Request-Header|Cache-Control|max-age=0
Request-Header|Connection|keep-alive
Request-Header|Content-Length|256
Request-Header|Content-Type|multipart/form-data; boundary=----WebKitFormBoundary46EP6zTN86hbbaJC
Request-Header|Cookie|visited=yes
Request-Header|DNT|1
Request-Header|Host|localhost:9090
Request-Header|Origin|http://localhost:9090
Request-Header|Referer|http://localhost:9090/form.html
Request-Header|Upgrade-Insecure-Requests|1
Request-Header|User-Agent|Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.162 Safari/537.36
Parts-Count|2
Part-ContainsContents|user|joe
Part-ContainsContents|comment|this is a simple comment